src/interfaces/IBPool.sol

```diff
@@ -46,6 +113,16 @@ interface IBPool is IERC20 {
     uint256 maxPrice
   ) external returns (uint256 tokenAmountOut, uint256 spotPriceAfter);
 
+  /**
+   * @notice Swaps as many tokens in as possible for an exact amount of tokens out
```

Comment:
`as many tokens in as possible` wrong in many ways

Reply:
Explain? I have an opinion very similar to this [comment](https://github.com/defi-wonderland/balancer-v1-amm/pull/64#discussion_r1628580613)